overlay.d/30gcp-udev-rules/usr/lib/dracut/modules.d/30gcp-udev-rules/module-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/bash
+# Install 64-gce-disk-removal.rules, 65-gce-disk-naming.rules and
+# google_nvme_id into the initramfs
+
+# called by dracut
+install() {
+    inst_simple google_nvme_id /usr/lib/udev/google_nvme_id
```

Comment:
Huh, I would've expected

```suggestion
    inst_simple /usr/lib/udev/google_nvme_id
```

here.

What you have clearly works (I don't see a dracut error in the compose) but I'm confused how. I would think this gets interpreted as asking dracut to install the executable `google_nvme_id` into `/usr/lib/udev/google_nvme_id` but `google_nvme_id` isn't in the `PATH` so it shouldn't find it. Can you sanity-check that the file indeed ends up in there? Alternatively, maybe cleaner to just use the simpler invocation? (But if you've already verified this, then feel free to ship as is).

Reply:
Yeah it works in the end:
```
$ ls -ls /usr/lib/udev/google_nvme_id
12 -rw-r--r--. 2 root root 9186 Jan  1  1970 /usr/lib/udev/google_nvme_id
```
But let me do it this way to be simpler